### DISCUSSION TITLE
Как закрыть доступ пользователю на изменение пароля?

### DISCUSSION THREAD

**Discussion by vldekov** [totumonline/totum-ru-issues-and-discussions#445] [Как это сделать?] [answered]
Есть возможность скрыть у конкретного пользователя в меню "Изменить пароль" ?
![image](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/114401808/948beb97-8588-4f29-9b1e-e3f0f45cb083)

**Comment by erpic-ru** [comment, marked as answer]
В Системных таблицах - настройках (там где логотип) найдите поле "Кнопки упраления для пользователей" и в его коде форматирования пропишите условие отображения для первой кнопки

**Reply by vldekov** [reply]
Спасибо!

**Reply by erpic-ru** [reply]
Возможно, при обновлении Тотума изменения сбросятся, так что либо добавьте поле в исключения. Либо прописывать придется каждый раз

**Reply by ttm-alexeypolunin** [reply]
В PRO смотрите Удерживаемые поля

https://ru.docs.totum.online/update#held-fields
